packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -784,8 +822,13 @@ class Navigator extends StatefulWidget {
   /// ```
   /// {@end-tool}
   @optionalTypeArgs
-  static Future<T> pushReplacementNamed<T extends Object, TO extends Object>(BuildContext context, String routeName, { TO result }) {
-    return Navigator.of(context).pushReplacementNamed<T, TO>(routeName, result: result);
+  static Future<T> pushReplacementNamed<T extends Object, TO extends Object>(
```

Comment:
Here and elsewhere:  instead of `TO` maybe `TResult`

Reply:
I'll leave this on as-is for now since it's not at the core of this PR.